chapters/annotations.tex

```diff
@@ -2324,6 +2324,18 @@ \subsection{Licensing}\label{licensing}
 \end{example}
 
 
+\section{License Texts}\label{license-texts}
+
+For a top-level class the {\lstinline!annotation(License="modelica:/ModelicaLibraryName/Resources/Licenses/MyLicense.txt")!}\annotationindex{License}, gives a license file for the class.
+The annotation may give a scalar value or an array of values.
+The license files are human-readable files containing the license conditions.
+If the annotation specifies an array of license files the conditions in all of them must be satisfied (i.e., it is not intended for dual-licensing).
+External functions may also contain this annotation (with similar content), see \cref{annotations-for-external-libraries-and-include-files}.
+When generating a distributable version for a translated model the license files for used functions and top-level classes should be included.
+Exceptions could be that licenses will be distributed in another way, either the same ones -- or there is an agreement to use different licenses (e.g., for internal libraries).
+Note that the collection of license files is only intended as a help, thus there is no need for a strict definition of distributable version or stricter rules for including the license files.
```

Comment:
I'm trying to see for whom this help is… Am I getting it wrong if I believe this is what is meant?
```suggestion
Note that the \lstinline!License!-annotation only allows the library developer to assist the tool in doing what is legally required; in the end it is a user responsibility to ensure that required license files accompany distributable versions of models.
```

I'd like to skip the part about _strict definition_, since I personally would find it helpful if the specification made it a tool responsibility in at least some of the most common situations, so that I would have the specification on my side when complaining to my tool vendor until they implement this.

Reply:
Turned into suggestion below.